lib/cocoapods/target.rb

```diff
@@ -46,6 +46,10 @@ class Target
     attr_reader :build_type
     private :build_type
 
+    # @return [Boolean] whether the target .
+    #
+    attr_accessor :application_extension_api_only
```

Comment:
we've tried hard to make those `attr_reader` instead of setting those fields at random places in the code. Can you update that to be an `attr_reader`?